nobrainer/metrics.py

```diff
@@ -132,3 +122,11 @@ def tversky(y_true, y_pred, axis=(1, 2, 3), alpha=0.3, beta=0.7):
     )
     # Sum over classes.
     return tf.reduce_sum((num + eps) / (den + eps), axis=-1)
+
+
+def dice_coef_multilabel(y_true, y_pred):
+    n_classes = tf.shape(y_pred)[-1]
+    dice_coeff = 0
+    for index in range(n_classes):
+        dice_coeff -= dice(y_true[:, :, :, :, index], y_pred[:, :, :, :, index])
```

Comment:
The multiple slices can be simplified with an ellipsis `...`

```suggestion
        dice_coeff -= dice(y_true[..., index], y_pred[..., index])
```